.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    node: true,
+  },
+  parserOptions: {
+    parser: 'babel-eslint',
+  },
+  extends: [
+    '@nuxtjs',
+    'prettier',
+    'prettier/vue',
+    'plugin:prettier/recommended',
+    'plugin:nuxt/recommended',
+  ],
+  plugins: ['prettier'],
+  // add your custom rules here
+  rules: {
+    'no-console': 'off',
+    'vue/no-v-html': 'off',
+    'vue/comment-directive': 0,
+  },
+  ignorePatterns: ['functions'],
+}
```

Comment:
Add new line

Reply:
@VOlekh why you have all your comments doubled?